### ISSUE TITLE
List if tutor is online or offline on the tutors page

### ISSUE THREAD

**Issue by grantal** [reed-college/paneity#116]
django-private-chat has a notion of whether or not a user is online or offline. So if a tutor is on the tutorchat page and waiting for a message, they will be denoted as online and if you go into a chat with them, it will say they are online at the top of the page.

So, the *tutors* page should list next to each tutor whether they are online or offline in this sense.

Maybe also sort the list so online tutors are at the top, but we can make a separate issue for that once this issue is done.

**Comment by grantal** [comment]
Probably gonna need to base something off of this function: https://github.com/Bearle/django-private-chat/blob/dev/django_private_chat/handlers.py#L64

**Comment by grantal** [comment]
It looms like django-prvate-chat only lets you check if another user is online if you have created a dialog with that user. I can see two ways of adding online/offline status to the tutors page:
1. When a student accesses the tutors page, the page creates a dialog with every tutor in the background
2. We modify django-private-chat so that you can check users online/offline status without having a dialog with them

Approach 1 is bad because it is very time inefficient and then the 'You are a tutor' page will clogged up with conversations that don't exist and viewing old messages through that page will become a nightmare.

So approach 2 it is. I will wait a while before implementing this because I don't want to add more complication to the setup as happened with spirit and also it will take me a while to figure out how to write server side websockets code.

**Comment by grantal** [comment]
I think I'm gonna go with approach 3: use ajax and sidestep all of this django-private-chat code

**Comment by grantal** [comment]
Okay 4th idea: Make a single user that has a dialog with every tutor and use that user to get the online or offline status on the tutors page

**Comment by grantal** [comment]
This was mostly completed with #257
